hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -168,12 +168,37 @@ registers: [
       },
     },
 
+    { multireg: {
+        cname: "SENSOR_CTRL",
+        name: "ALERT_EN",
+        regwen: "CFG_REGWEN",
+        desc: '''
+          Each bit enables a corresponding alert.
+        ''',
+        count: "NumAlertEvents",
+        swaccess: "rw",
+        hwaccess: "hro",
+        fields: [
+            { bits: "0",
+              name: "VAL",
+              desc: '''
+                1 - An alert event is enabled.
+                0 - An alert event is disabled.
+              ''',
+              resval: "0"
+            },
+        ],
```

Comment:
Would it make sense to use MuBi here (to protect the value written by SW and at rest in the register)? I noted that `FATAL_ALERT_EN` isn't MuBi, but the `ALERT_EN` register has even more impact

Reply:
I can make this a mubi, yes.

Reply:
Ok I have converted all of these to MuBi4 registers.